lib/scripts/oneoff/backup/populate-db-09-13-21.ts

```diff
@@ -0,0 +1,260 @@
+/* eslint-disable */
+import csv from 'csv-parser'; // CSV parser
+import fs from 'fs'; // File system
+import {
+  ApplicantStatus,
+  ApplicationStatus,
+  Gender,
+  PaymentType,
+  PhysicianStatus,
+  Province,
+} from '@lib/graphql/types'; // GraphQL types
+import {
+  UpsertApplicant,
+  UpsertApplication,
+  UpsertApplicationProcessing,
+  UpsertGuardian,
+  UpsertMedicalInformation,
+  UpsertPermit,
+  UpsertPhysician,
+} from '@prisma/types'; // Seeding types
+
+// First ID of records to insert
+const STARTING_INDEX = 10;
+
+// Type of JS object parsed from CSV row
+type CsvRow = {
+  userid: string;
+  permit_before: string;
+  initial_apply_org: string;
+  lastname: string;
+  firstname: string;
+  address: string;
+  city: string;
+  prov: string;
+  postal_code: string;
+  telephone: string;
+  email_1: string;
+  email_2: string;
+  DOB: string;
+  gender: string;
+  application_date: string;
+  permit_no: string;
+  fee: string;
+  donation: string;
+  pay_method: string;
+  disability: string;
+  eligibility: string;
+  certification_date: string;
+  'permit expiry': string;
+  receipt: string;
+  'guardian-lastame': string;
+  'guardian-firstName': string;
+  'guardian-address': string;
+  'guardian-unit': string;
+  'guardian-telphone': string;
```

Comment:
is it supposed to be 'guardian-telephone' ?

Reply:
Sorry I had to merge quickly! These may seem like typos but they are typos in the CSV file RCD provided. I wanted to keep the CSV file intact so that we have a consistent source of truth which is why we have the typos in the review comments you left.